dubbo-common/src/main/java/org/apache/dubbo/common/utils/ReflectUtils.java

```diff
@@ -1273,16 +1272,18 @@ public static boolean isPublicInstanceField(Field field) {
         Class<?> beanClass = bean.getClass();
         BeanInfo beanInfo = null;
         T propertyValue = null;
+
+        beanClass.getDeclaredFields();
```

Comment:
Why invoke the getDeclaredFields method here ?

Reply:
It's useless, will remove it now.